views-core/src/main/java/io/micronaut/views/ViewsRenderer.java

```diff
@@ -19,15 +19,15 @@
 import io.micronaut.core.annotation.Nullable;
 import io.micronaut.core.io.Writable;
 import io.micronaut.core.order.Ordered;
-import io.micronaut.http.HttpRequest;
 
 /**
  * Interface to be implemented by View Engines implementations.
  * @param <T> The model type
+ * @param <R> The response type
```

Comment:
should this be request instead of response?
```suggestion
 * @param <R> The request type
```